lampod/src/ln/inventory_manager.rs

```diff
@@ -32,10 +32,12 @@ impl InventoryHandler for LampoInventoryManager {
 
         match event {
             InventoryCommand::GetNodeInfo(chan) => {
+                let chain = self.channel_manager.conf().network.to_string();
```

Comment:
```suggestion
                let chain = self.channel_manager.conf.network.to_string();
```

if you can not do that, try `pub (crate) conf`